src/App.js

```diff
@@ -56,13 +58,22 @@ const App = () => {
         })}
         {shouldDisplayScore && (
           <Jumbotron>
-            <h2> You have got {score} out of {data.length} </h2>
+            <h2>
+              {' '}
+              You have got {score} out of {data.length}{' '}
```

Comment:
It looks like Prettier formatted this so that the text content of `h2` is on a separate line (it's good that prettier is reformatting). However, it looks like the spaces at the beginning and end of the text (which I missed in code review) were converted into `{' '}` which is unnecessary, so both `{' '}`s can be removed. Just in case you don't already do it, it's always a good idea to review your code changes before committing to see if there's any code that can be tidied up 👍

Reply:
Oh, I missed it! Removed